kibble/datastore/film_index_datasource.go

```diff
@@ -38,8 +39,11 @@ func (ds *FilmIndexDataSource) GetEntityType() reflect.Type {
 // Iterator - return a list of all films, iteration of 1
 func (ds *FilmIndexDataSource) Iterator(ctx models.RenderContext, renderer models.Renderer) (errCount int) {
 	clonedFilms := make([]*models.Film, len(ctx.Site.Films))
+
+	i := 0
 	for _, f := range ctx.Site.Films {
-		clonedFilms = append(clonedFilms, transformFilm(*f))
+		clonedFilms[i] = transformFilm(*f)
```

Comment:
I don't think you need a separate `i`.

`for i, f := range ctx.Site.Films` should work fine right?

Reply:
It used to work. Now its a map `i` and `f` correspond to key and value. `i` being `"/film/123"`

Reply:
Ahh true!